src/us/kbase/workspace/database/Workspace.java

```diff
@@ -1025,44 +1026,46 @@ public UserWorkspaceIDs listWorkspaceIDs(
 		return db.getObjectInformation(params.resolve(pset));
 	}
 	
+	/** Get data objects from the workspace.
+	 * @param user the user requesting the objects, or null for an anonymous user.
+	 * @param objs the requested objects; no more than 10,000.
+	 * @param noData true to return information about the object only, no data. E.g.
+	 * {@link WorkspaceObjectData#getSerializedData()} will be empty.
+	 * @param nullIfInaccessible instead of throwing an exception, if any object is not accessible
+	 * for any reason, including deletion, include a null in the returned list at its position.
+	 * @param asAdmin run this method as an admin, bypassing permissions checks.
+	 * @return the data objects.
+	 * @throws InaccessibleObjectException if any of the objects are inaccessible.
+	 * @throws NoSuchObjectException if a requested object doesn't exist.
+	 * @throws NoSuchReferenceException if a reference path is invalid.
+	 * @throws ReferenceSearchMaximumSizeExceededException if a valid path to an object could not
+	 * be found within the specified maximum size of the graph search.
+	 * @throws TypedObjectExtractionException if extracting a data subset from a data object
+	 * fails.
+	 * @throws WorkspaceCommunicationException if a communication error occurs when contacting the
+	 * storage system.
+	 * @throws CorruptWorkspaceDBException if corrupt data is found in the storage system.
+	 */
 	public List<WorkspaceObjectData> getObjects(
 			final WorkspaceUser user,
-			final List<ObjectIdentifier> loi)
-			throws CorruptWorkspaceDBException,
-				WorkspaceCommunicationException, InaccessibleObjectException,
-				NoSuchReferenceException, TypedObjectExtractionException,
-				ReferenceSearchMaximumSizeExceededException, NoSuchObjectException {
-			
-		return getObjects(user, loi, false);
-	}
-	
-	public List<WorkspaceObjectData> getObjects(
```

Comment:
These two helper methods only support deprecated methods, so I got rid of them and just had the deprecated methods call the main version of the method below